src/components/BounceActivityContainer/index.scss

```diff
@@ -0,0 +1,32 @@
+.container {
```

Comment:
let's remember to make classnames more BEM-like for container, h1 -> to be less specific in selecting all h1s but rather certain classes, loading such that they don't conflict with style-guide classnames which also use container and loading